app/api/dao/user_extension.py

```diff
@@ -21,17 +21,50 @@ def create_user_extension(data):
                 A dictionary containing "message" which indicates whether or not the user_exension was created successfully and "code" for the HTTP response code.
         """
 
-        user_id = data["user_id"]
-        is_organization_rep = data["is_organization_rep"]
+        user_id = AUTH_COOKIE["user_id"].value
         timezone = data["timezone"]
+        
+        additional_info_data = {
```

Comment:
Keep this as a default dict instead of initializing with empty strings

Reply:
done. now default dict {}